config/samples/config_v1alpha1_verifier_notary_certstore.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: config.ratify.deislabs.io/v1alpha1
+kind: Verifier
+metadata:
+  name: verifier-notary
+spec:
+  name: notaryv2
+  artifactTypes: application/vnd.cncf.notary.signature
+  parameters:
+    verificationCertStores:
+      certs:
+          - certstore-akv
+          - certstore-akv1
+      certs1:
+          - certstore-akv2
+          - certstore-akv3
+    trustPolicyDoc:
+      version: "1.0"
+      trustPolicies:
+        - name: default
+          registryScopes:
+            - "*"
```

Comment:
As discussed, please add in documentation on what '*' here means.

Reply:
sure, in the latest version included a link to Trust policy docs